### PR TITLE
ODM-9555 update docs for reference genome upload

### PR DESCRIPTION
**Pull request by olepark-gs**

### CODE REVIEW

File: genestack_client/data_importer.py [hunk 1]
```diff
@@ -485,17 +485,17 @@ def create_reference_genome(self,
                                 parent=None,
                                 name=None,
                                 description='',
-                                sequence_urls=None,
+                                sequence_urls=[],
```
Comment (klas-genestack) [inline genestack_client/data_importer.py:488]: [Never use a mutable default argiment](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).
Also, why this change at all?

Comment (olepark-gs) [reply genestack_client/data_importer.py]: Fixed. The idea here is that sequence files are no longer used in reference genome related functionality. I updated the docs, removed redundant metadata addition and left the parameter untouched for backwards compatibility. Also added an example script that can be used to add genomes

**Review by dgm90 — APPROVED** [review]

File: genestack_client/data_importer.py [hunk 2]
```diff
@@ -508,7 +508,7 @@ def create_reference_genome(self,
         :type name: str
         :param description: experiment description
         :type description: str
-        :param sequence_urls: list urls or local path to sequencing files.
+        :param sequence_urls: list urls or local path to sequencing files. (deprecated, unused)
```
Comment (klas-genestack) [inline genestack_client/data_importer.py:511]:
```suggestion
        :param sequence_urls: [OBSOLETE] list urls or local path to sequencing files
```

File: genestack_client/scripts/upload_reference_genome.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3.9
```
Comment (klas-genestack) [inline genestack_client/scripts/upload_reference_genome.py:1]: Why specific Python version?  Also, Python Client is still supposed to work with Python 2.7, so I don't think it's acceptable to have a Python 3-only script here

**Review by klas-genestack — CHANGES_REQUESTED** [review]
Why `genestack_client/scripts/upload_reference_genome.py` should be in this repo at all?

File: genestack_client/scripts/upload_reference_genome.py (hunk 3 above)
Comment (olepark-gs) [reply genestack_client/scripts/upload_reference_genome.py:1]: I will comment regarding both this and in general why this script is in this repo. We need to simplify the process of reference genome upload to ODM, so I needed to introduce a reusable example of how to upload new genomes. Do you think it doesn't fit this repo? How should it be distributed to clients at all? Maybe they don't need such example?
There is a second reason I put it here - I wanted to reuse it in future to upload genomes at ODM init. Now there's a copy of this script in unified repo, and I want to eventually get rid of it in favour of import from python client

Comment (olepark-gs) [reply genestack_client/scripts/upload_reference_genome.py:1]: The version is stated like this, again, because it was just copied as is from the unified repo. I will test it using python 2.7 to make sure it works for that version as well, if it should be compatible with both

**Review by genestack-okunitsyn — APPROVED** [review]

**Review by klas-genestack — APPROVED** [review]
